notebooks/Calculate_sensitivity_eventio.ipynb

```diff
@@ -133,7 +133,11 @@
   },
   {
    "cell_type": "code",
+<<<<<<< HEAD
    "execution_count": 8,
+=======
+   "execution_count": 10,
+>>>>>>> 71399bf6e2af2638cd48a61813380c1583a879ea
    "metadata": {},
```

Comment:
Please clean-up the notebook
ps: it does not seem that you made changes to the notebook that need to be pushed, maybe you just want to completely remove it from the PR

Reply:
yes I will remove it, I thought I added only my new script in this PR.